x-pack/apm-server/sampling/pubsub/pubsubtest/client.go

```diff
@@ -106,20 +106,22 @@ func (rt *channelClientRoundTripper) RoundTrip(r *http.Request) (*http.Response,
 	var handler func(*http.Request, *httptest.ResponseRecorder) error
 	switch r.Method {
 	case "GET":
-		if strings.HasSuffix(r.URL.Path, "/_stats/get") {
+		if strings.HasSuffix(r.URL.Path, "/") {
```

Comment:
I think this should be `r.URL.Path == "/"` , rather than `strings.HasSuffix`, to avoid matching for example, `/_stats/get/`

Reply:
I remember now why I did that, and everywhere else is doing `strings.HasSuffix`: r.URL.Path is always returning (and I have no idea why) host+path, eg.:

```
testing.invalid/
testing.invalid/traces-sampled-testing/_stats/get
testing.invalid/index_name/_refresh
testing.invalid/index_name/_search
```

We're only using `testing.invalid/`, and I can do that as a direct comparison